pkg/scheduler/framework/plugins/dynamicresources/dynamicresources.go

```diff
@@ -491,6 +493,33 @@ func (pl *dynamicResources) isSchedulableAfterClaimChange(logger klog.Logger, po
 	return framework.Queue, nil
 }
 
+// isSchedulableAfterResourceSliceChange is invoked for add and update slice events reported by
+// an informer. It checks whether that change made a previously unschedulable
```

Comment:
Explain what exactly could make it schedulable according to this function.

It looks like we just care whether the pod claims exist, but we don't check any correlation between the claims and the ResourceSlice

Reply:
Updated.

We had this discussion on Slack (?) about the tradeoff between execution time for queuing hints vs. accuracy. Based on that discussion, I'm erring on the side of faster execution.